__tests__/app.test.js

```diff
@@ -48,4 +49,34 @@ describe('/api (getting all endpoints)', () => {
         });
     });
 });
+
+describe('/api/articles/:article_id', () => {
+    test('making sure that its responding with an article with the desired id', () => {
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+        .then((article) => {
+            expect(article.body.article.article_id).toBe(2);
+        });
+    });
```

Comment:
we should also check that the article has all the desired properties that we would expect it to have